### PR TITLE
Add task to disable prelinking

### PR DESCRIPTION
**Pull request by hooksie1**
Task disables prelinking using lineinfile.

### CODE REVIEW

**Comment by shawndwells** [comment]
Thanks for the contribution! A few small tweaks are needed:

1. The build system links things together using the XCCDF rule ID as the key. In this case, the rule name is:
``<Rule id="disable_prelink" prodtype="rhel7">``
Ref: https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/xccdf/system/software/integrity.xml#L13

Switch the name of the ansible script to disable_prelink.yml and the build system will integrate it.

2. SCAP allows for metadata on remediations -- things like "is a reboot required?". That metadata is prepended to ansible scripts like this:
`````
# platform = multi_platform_rhel,multi_platform_fedora
# reboot = false
# strategy = restrict
# complexity = low
# disruption = low
`````
https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/templates/static/ansible/accounts_minimum_age_login_defs.yml#L1#L5

3. This remediation can be used beyond RHEL7, so lets move it into the shared directory:
``$ git mv RHEL/7/templates/static/ansible/prelink-disabled.yml shared/templates/static/ansible/``

**Comment by hooksie1** [comment]
Ok I'll go through and make these changes. I'll do the same on the other pull requests I just made.

**Comment by dahaic** [comment]
Jenkins test this please.

File: shared/templates/static/ansible/disable_prelink.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: disable prelinking
+  lineinfile:
+    path: /etc/sysconfig/prelink
+    regexp: '^PRELINK'
```
Comment (yuumasato) [inline shared/templates/static/ansible/disable_prelink.yml]: It is not a big deal, but it would be better to have regexp like `^PRELINKING=`

Comment (hooksie1) [reply shared/templates/static/ansible/disable_prelink.yml]: Ok changed regex.

**Comment by yuumasato** [comment]
Cool, thank you for the patch.

**Pull request merged**